spread/runner.go

```diff
@@ -1073,6 +1081,44 @@ func (s *stats) log() {
 	logNames(printf, "Failed project restore", s.ProjectRestoreError, projectName)
 }
 
+func (s *stats) addTestsToReport(report Report, testsList []*Job, result string, verb string) {
+	for _, job := range testsList {
+		splittedName := strings.Split(taskName(job), "/")
+		suiteName := strings.Join(splittedName[:len(splittedName)-1], "/")
+		testName := splittedName[len(splittedName)-1]
```

Comment:
```suggestion
		split := strings.Split(taskName(job), "/")
		suiteName := strings.Join(split[:len(split)-1], "/")
		testName := split[len(split)-1]
```
looks like this could have been done using strings.LastIndex() or filepath.Dir/Base() if the test name always look like paths, maybe something like:
```
// task names look like paths (eg suite/foo/bar/job)
name := taskName(job)
suiteName := filepath.Dir(name)
testName := filepath.Base(name)
```

Reply:
Makes sense, I'll do the change